File: module-contracts/src/main/java/com/beat/contracts/booking/readmodel/MakerTicketListItemReadModel.java
package com.beat.contracts.booking.readmodel;

import java.time.LocalDateTime;

import com.beat.contracts.common.ReadModel;

@ReadModel
public record MakerTicketListItemReadModel(
	Long bookingId,
	String bookerName,
	String bookerPhoneNumber,
	Long scheduleId,
	int purchaseTicketCount,
	LocalDateTime createdAt,
	MakerTicketBookingStatus bookingStatus,
	String bankName,
	String accountNumber,
	String accountHolder
) {
}

File: module-contracts/src/main/kotlin/com/beat/contracts/booking/readmodel/MakerTicketListItemReadModel.kt
package com.beat.contracts.booking.readmodel


import com.beat.contracts.common.ReadModel
import java.time.LocalDateTime

@ReadModel
data class MakerTicketListItemReadModel(
    val bookingId: Long,
    val bookerName: String,
    val bookerPhoneNumber: String,
    val scheduleId: Long,
    val purchaseTicketCount: Int,
    val createdAt: LocalDateTime,
    val bookingStatus: MakerTicketBookingStatus,
    val bankName: String,
    val accountNumber: String,
    val accountHolder: String,
)
